tutor/src/components/annotations/annotation.jsx

```diff
@@ -157,27 +149,27 @@ export default class AnnotationWidget extends React.Component {
   @observable activeHighlight = null;
   @observable widgetStyle = null;
   @observable needToGetReferenceElements = true;
-  @observable showWindowShade = false;
-  @observable allAnnotationsForThisBook = [];
+  @computed get showWindowShade() {
+    return User.annotations.ux.isSummaryVisible;
+  }
+
   @observable parentRect = {};
   @observable referenceElements = [];
 
   @computed get annotationsForThisPage() {
-    setTimeout(() => this.forceUpdate(), 1000);
-    return this.allAnnotationsForThisBook.filter((item) =>
+    return filter(this.allAnnotationsForThisBook, (item) =>
```

Comment:
You could use the built-in Array `filter` method instead of the lodash function.

Reply:
yeah, welcome to our debate!  Since `lodash` is already used pervasively through the code base and is generally faster, we're pretty fond of using it instead of the "builtin" functions.  

For instance https://jsperf.com/array-prototype-filter-vs-filter shows lodash being %2.2 faster than `.filter` on chrome.

Which isn't really enough speedup to bother with, and I normally wouldn't have changed the line to use lodash.  In fact I'm not sure why I did here.  

I think it might be because I'd originally used lodash's object search and had this as: `filter(this.allAnnotationsForThisBook, { chapter, section })`

Then you added the elementId clause and I rebased it to the original and didn't change the `filter` back.